File: ankey-docker-runner/src/main/java/io/github/uniclog/docker/runner/Activator.java
package io.github.uniclog.docker.runner;

import com.intellij.ide.AppLifecycleListener;
import com.intellij.openapi.application.ApplicationManager;
import com.intellij.openapi.ui.Messages;

public class Activator implements AppLifecycleListener {
    @Override
    public void appStarted() {
        System.out.println(">>> Docker Runner started");
        var manager = ApplicationManager.getApplication();
        manager.invokeLater(() -> Messages.showMessageDialog(
                "Docker Runner  started!",
                "Docker Runner",
                Messages.getInformationIcon()
        ));
    }
}

File: ankey-docker-runner/src/main/java/io/github/uniclog/docker/runner/Activator.kt
package io.github.uniclog.docker.runner

import com.intellij.ide.AppLifecycleListener
import com.intellij.openapi.application.ApplicationManager
import com.intellij.openapi.ui.Messages

class Activator : AppLifecycleListener {
    override fun appStarted() {
        println(">>> Plugin B started")
        ApplicationManager.getApplication().invokeLater {
            Messages.showMessageDialog(
                "Plugin B started!",
                "Plugin B",
                Messages.getInformationIcon()
            )
        }
    }
}